### PR TITLE
Replace outdated questions img

### PR DESCRIPTION
**Pull request by tferdous17**
Replaces the current questions img on the site with a new & updated one that displays the new Q&A system which utilizes threads

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Zabuzard) [inline .idea/misc.xml:1]: why this change?

File: website/src/main/resources/public/index.html [hunk 2]
```diff
@@ -92,7 +92,7 @@ <h1><span class="image right"><img style="width: 50%" src="images/logo_frame.png
     <!-- One -->
     <section id="one" class="wrapper style2 spotlights">
         <section>
-        <span class="image"><img src="images/questions.png" alt="Questions and Answers"
```
Comment (Zabuzard) [inline website/src/main/resources/public/index.html:95]: please keep the old image name and replace it instead of adding a new image, thanks

**Review by Zabuzard — CHANGES_REQUESTED** [review]

File: .idea/misc.xml (hunk 1 above)
Comment (tferdous17) [reply .idea/misc.xml:1]: Will revert it back to the original, I believe the misc.xml file automatically updated itself when I was working on changes in the other files.

**Review by Zabuzard — APPROVED** [review]
